backend/canisters/user_index/api/src/queries/mod.rs

```diff
@@ -1,6 +1,6 @@
-pub mod c2c_is_super_admin;
 pub mod c2c_lookup_principal;
 pub mod c2c_lookup_user_id;
+pub mod c2c_lookup_user_id_v2;
```

Comment:
c2c_lookup_user ?

Reply:
Done. We can consolidate the usages of the other 2 methods as a follow on commit.